lib/SPIRV/libSPIRV/SPIRV.debug.h

```diff
@@ -499,12 +499,14 @@ enum {
 
 namespace TypeInheritance {
 enum {
-  ChildIdx     = 0,
-  ParentIdx    = 1,
-  OffsetIdx    = 2,
-  SizeIdx      = 3,
-  FlagsIdx     = 4,
-  OperandCount = 5
+  ChildIdx        = 0,
```

Comment:
I thought there was no 'child'?

Reply:
Ultimately we want to do a refactoring splitting a single DebugInfo class to several:
```
DebugInfoCommon -> DebugInfoOpenCL
                                 -> DebugInfoNonSemantic -> Shader100
                                                                              -> Shader200
```
it will simplify future support

Reply:
Unification sounds good, although it would increase the amount of duplicating code (if we talk only about namespaces).
As I remember, it's not the first change that uses offset instead of separate namespace. But if we are about to unify and refactor such things, I believe it can be fixed in a separate PR (I can also help with that)